<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardWidget</class>
 <widget class="QWidget" name="CardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>65</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#CardWidget:hover {
border: 0px solid #EAF1F8;
border-radius: 8px;
background-color: #EAF1F8;
}
#cardCode, #cardStatus {
color: #415982;
font-family: Roboto, Helvetica;
font-size: 12px;
font-weight: 400;
}
#cardName {
color: #003168;
font-size: 16px;
font-weight: 700;
}</string>
  </property>
  <layout class="QGridLayout" name="CardWidgetLayout" columnstretch="0,0,1">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="cardName">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">MARI MAASIKAS MUSTIKAS</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cardCode">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">48405050123 |</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSvgWidget" name="cardIcon" native="true">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="cardStatus">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>ID-card</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">QSvgWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
